src/settings/components/utils.ts

```diff
@@ -496,13 +496,32 @@ export function defaultAsFirstOrderIdentities(identities: Array<Identity>): Arra
 	return result;
 }
 
+/**
+ * Return the requested attribute value as an array if present, or an empty array otherwise.
+ *
+ * @param settings
+ * @param key
+ * @returns
+ */
+export function getAttributeIfPresent(settings: AccountSettings, key: string): Array<string> {
+	const attributeValue = settings.attrs[key];
+	if (attributeValue) {
+		if (isArray(attributeValue)) {
+			return [...(attributeValue as string[])];
```

Comment:
there's no need to spread attributeValue to create the array, since it is already.